tests/test_profile.py

```diff
@@ -235,3 +235,50 @@ def test_that_links_in_the_services_page_return_200_code(self, base_url, seleniu
                 bad_urls.append(check_result)
 
         assert 0 == len(bad_urls), u'%s bad links found. ' % len(bad_urls) + ', '.join(bad_urls)
+
+    @pytest.mark.credentials
+    @pytest.mark.nondestructive
+    def test_that_user_can_view_external_accounts(self, base_url, selenium, vouched_user):
+        home_page = Home(base_url, selenium)
+        home_page.login(vouched_user['email'], vouched_user['password'])
+        settings = home_page.header.click_settings_menu_item()
+
+        irc_form = settings.external_accounts.irc_form
+        external_accounts_form = settings.external_accounts.external_accounts_form
+        assert irc_form
+        assert external_accounts_form
+
+    @pytest.mark.credentials
+    def test_that_user_can_add_external_account(self, base_url, selenium, vouched_user):
+        home_page = Home(base_url, selenium)
+        home_page.login(vouched_user['email'], vouched_user['password'])
+        settings = home_page.header.click_settings_menu_item()
+
+        external_accounts_form = settings.external_accounts.external_accounts_form
+        cnt_external_accounts = external_accounts_form.count_external_accounts()
+        external_accounts_form.click_add_account()
+        new_cnt_external_accounts = external_accounts_form.count_external_accounts()
+        assert new_cnt_external_accounts == (cnt_external_accounts + 1)
```

Comment:
Nit: We tend to use `assert <expected> <actual>`, so you'll need to switch these around.